pyproject.toml

```diff
@@ -154,3 +154,7 @@ module = [
   "securesystemslib.*",
 ]
 ignore_missing_imports = "True"
+
+[tool.pydocstyle]
+inherit = false
+ignore = "D400,D209,D415,D213,D205,D202,D107,D407,D413,D212,D104,D406,D105,D411,D401,D200,D203"
```

Comment:
This changes with #2281 #2282, right?

Reply:
Yes, reduce the listing over time. In the mean time, not to introduce new violations.